### PR TITLE
Fix execution unit tests on linux mint

### PR DESCRIPTION
**Pull request by wbarcovsky**
I run unit tests on my local machine with linux mint v19.2 and all tests with mongodb adapter fails.
Turns out it was an error in package "mongodb-memory-server": https://github.com/nodkz/mongodb-memory-server/issues/32
It was fixed in the version 6.1.0: https://github.com/nodkz/mongodb-memory-server/releases/tag/v6.1.0

In this PR I upgrade version of this package up to 6.1.1 to fix this

### CODE REVIEW

File: packages/mongo-join-builder/package.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
     "@keystonejs/utils": "^5.1.0",
     "cuid": "^2.1.6",
-    "lodash.omitby": "^4.6.0"
+    "lodash.omitby": "^4.6.0",
+    "mongodb-memory-server": "6.1.1"
   },
   "devDependencies": {
-    "mongodb": "^3.3.3",
-    "mongodb-memory-server": "^6.0.1"
+    "mongodb": "^3.3.3"
```
Comment (timleslie) [inline packages/mongo-join-builder/package.json]: Is there a particular reason for moving this out of `devDependencies`?

Comment (wbarcovsky) [reply packages/mongo-join-builder/package.json]: My bad, will fix it ASAP

**Comment by timleslie** [comment]
This all looks good, as a final step could I get you to add a `changeset`?  (run `yarn changeset` and follow the instructions there). Thanks!

**Comment by wbarcovsky** [comment]
> This all looks good, as a final step could I get you to add a `changeset`? (run `yarn changeset` and follow the instructions there). Thanks!

Done!

**Pull request merged**